src/main/java/org/jabref/gui/contentselector/ContentSelectorDialog.java

```diff
@@ -302,8 +310,7 @@ private void applyChanges() {
 
         // Update all selectors in the current BasePanel.
         if (changedFieldSet) {
-            // We have added or removed content selectors, update the entry editor
-            panel.rebuildAllEntryEditors();
+            // TODO: We have added or removed content selectors, update the entry editor
```

Comment:
I have this in mind. I plan to implement content selectors along with auto-completion in one of the next PRs (as soon as the big problems are resolved).